prompts/template_xml/triage_priming.txt

```diff
@@ -0,0 +1 @@
+Given the following crash report and fuzz driver, investigate the cause of the crash.
```

Comment:
Do we need to ask LLM to keep the response succinct?
Sometimes, I found it returned a wall of text with a lot of useless info.

Reply:
Yes, the problem you mentioned does exist. I tried to make the following modifications and asked LLM to give a concise answer. But at the same time, after the modification, in some cases LLM will answer too concisely. We may need better prompts in the future.

https://github.com/fdt622/oss-fuzz-gen/blob/0de21f16d9254cc7c4fff782276d48741bab9643/prompts/template_xml/triager_priming.txt#L3